npm package @cloudcomponents/cdk-codepipeline-slack@1.0.31 fails to run

After installing and configuring `@cloudcomponents/cdk-codepipeline-slack@1.0.31` (the latest stable version), lambda SNS listener fails to run with the error: 
```
{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module '@slack/web-api'",
    "stack": [
        "Runtime.ImportModuleError: Error: Cannot find module '@slack/web-api'",
        "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
        "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
        "    at Object.<anonymous> (/var/runtime/index.js:45:30)",
        "    at Module._compile (internal/modules/cjs/loader.js:778:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)",
        "    at Module.load (internal/modules/cjs/loader.js:653:32)",
        "    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:585:3)",
        "    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)",
        "    at startup (internal/bootstrap/node.js:283:19)"
    ]
```
If I repackage lambda code myself with all prod-scoped dependencies - it works fine

--- a/packages/cdk-codepipeline-slack-lambda/package.json
+++ b/packages/cdk-codepipeline-slack-lambda/package.json
@@ -9,8 +9,8 @@
         "watch": "tsc -w"
     },
     "bundleDependencies": [
-        "@slack/client",
         "@slack/interactive-messages",
+        "@slack/web-api",
         "aws-serverless-express",
         "express"
     ],

